README.md

```diff
@@ -32,7 +32,7 @@
 - [📝 License](#license)
 
 # 📖 [LeaderBoard ] <a name="about-project"></a>
```

Comment:
- [ ] This title doesn't correlate with the project name and what it does. Kindly fix this.